### PR TITLE
Add `is-cool.dev`, `is-local.org`, `is-not-a.dev` and `localplayer.dev`

### PR DESCRIPTION
**Pull request by wdhdev**
Public Suffix List (PSL) Pull Request (PR) Template
====

Each PSL PR needs to have a description, rationale, indication of DNS validation and syntax checking, as well as a number of acknowledgements from the submitter.  This template must be included with each PR, and the submitting party MUST provide responses to all of the elements in order to be considered.

### Checklist of required steps

* [x] Description of Organization
* [x] Robust Reason for PSL Inclusion
* [x] DNS verification via dig
* [x] Run Syntax Checker (make test)

* [x] Each domain listed in the PRIVATE section has and shall maintain at least two years remaining on registration, and we shall keep the \_PSL txt record in place in the respective zone(s) in the affected section

__Submitter affirms the following:__ 

  * [x] We are listing *any* third-party limits that we seek to work around in our rationale such as those between IOS 14.5+ and Facebook (see [Issue #1245](https://github.com/publicsuffix/list/issues/1245) as a well-documented example)

  * [x] This request was _not_ submitted with the objective of working around other third-party limits

  * [x] The [Guidelines](https://github.com/publicsuffix/list/wiki/Guidelines) were carefully _read_ and _understood_, and this request conforms
  * [x] The submission follows the [guidelines](https://github.com/publicsuffix/list/wiki/Format) on formatting and sorting

---

For Private section requests that are submitting entries for domains that match their organization website's primary domain, please understand that this can have impacts that may not match the desired outcome and take a long time to rollback, if at all.

To ensure that requested changes are entirely intentional, make sure that you read the affectation and propagation expectations, that you understand them, and confirm this understanding. 

PR Rollbacks have lower priority, and the volunteers are unable to control when or if browsers or other parties using the PSL will refresh or update.

(Link: [about propagation/expectations](https://github.com/publicsuffix/list/wiki/Guidelines#appropriate-expectations-on-derivative-propagation-use-or-inclusion))

 * [x] *Yes, I understand*.  I could break my organization's website cookies etc. and the rollback timing, etc is acceptable.  *Proceed*.
---

Description of Organization
====

Open Domains is a service of which you can register free subdomains on 4 different domains `is-cool.dev`, `is-local.org`, `is-not-a.dev` and `localplayer.dev`. We provide this service for completely free and users can register as many subdomains as they want.

I am an Admin for the Open Domains project.

Organization Website: 

https://open-domains.net
https://github.com/open-domains/register

Reason for PSL Inclusion
====

These domains (`is-cool.dev`, `is-local.org`, `is-not-a.dev` and `localplayer.dev`) should be included on the Public Suffix List since they *are* public suffixes available to everyone.

Some of the users' subdomains use name server records. These subdomains are unable to be used with services like Cloudflare as they are not classified as domains at the moment. We would like users to be able to use these certain services as most people use them.

One of the main reasons to be included is cookie separation as some of our domains use login services, etc.

This project is very similar of [js.org](https://github.com/js-org/js.org) and [eu.org](https://eu.org) of which are already on the public suffix list and we are providing the exact same services as them.

As every subdomain is seperate and is owned by different people we also need cookie separation as they use certain services that don't work well with other subdomains using the same service but on a different account.

Number of users this request is being made to serve:

Over 85. This is current.

P.S: You can see the number of domains we have [here](https://img.shields.io/github/directory-file-count/open-domains/register/domains?label=domains&style=for-the-badge&type=file). This is updated as soon as a new subdomain is registered. This number **does not** include the [reserved domains](https://github.com/open-domains/register/tree/main/domains/reserved).

DNS Verification via dig
=======

```
dig +short TXT _psl.is-cool.dev
"https://github.com/publicsuffix/list/pull/1672"
```

```
dig +short TXT _psl.is-local.org
"https://github.com/publicsuffix/list/pull/1672"
```

```
dig +short TXT _psl.is-not-a.dev
"https://github.com/publicsuffix/list/pull/1672"
```

```
dig +short TXT _psl.localplayer.dev
"https://github.com/publicsuffix/list/pull/1672"
```

Results of Syntax Checker (`make test`)
=========

All tests passed.

### CODE REVIEW

**Comment by andrewstech** [comment]
As the owner of Open-domains. I would like to point out that main reason for PSL inclusion is to help with super cookies. Any other side effects of the PSL are appreciated.

**Comment by WilliamDavidHarrison** [comment]
> As the owner of Open-domains. I would like to point out that main reason for PSL inclusion is to help with super cookies. Any other side effects of the PSL are appreciated.

In addition to this, most users are unable to use their subdomains with services such as Vercel and Netlify, since the domains are not recognised as public suffixes.

**Comment by WilliamDavidHarrison** [comment]
@dnsguru Is there an ETA on PR reviews? As it has been more than 2 months with no review.

**Comment by WilliamDavidHarrison** [comment]
@dnsguru @sleevi Is there an ETA on PR reviews? As it has been more than 3 months with no review.

**Comment by WilliamDavidHarrison** [comment]
@dnsguru Is there any ETA on this PR being reviewed? It's been more than 5 months...

**Comment by WilliamDavidHarrison** [comment]
I'm just gonna close this PR as it is clear that the PSL maintainers are not reviewing PRs anymore.

**Comment by dnsguru** [comment]
There is a queue ahead of this and wait times are what they are, but closing it or patience are the options available

**Comment by WilliamDavidHarrison** [comment]
I understand there are other PRs but even newer PRs than this are getting labelled and reviewed, meanwhile this PR has been open since **December 2022** with no response whatsoever.

**Comment by WilliamDavidHarrison** [comment]
@simon-friedberger I have fixed the sorting for the domains, I didn't realise they were first sorted by TLD, my bad.

**Comment by simon-friedberger** [comment]
@WilliamDavidHarrison You don't seem to have the required 2 years of remaining validity.

**Comment by WilliamDavidHarrison** [comment]
@simon-friedberger, @andrewstech has renewed all of the domains up to Janurary 2026.

**Review by simon-friedberger — APPROVED** [review]
LGTM

**Comment by WilliamDavidHarrison** [comment]
@simon-friedberger Is it possible for me to update the submission email on the PR (currently admin@open-domains.net) without having our position in the queue affected?

**Comment by dnsguru** [comment]
sounds good. there's big backlog on available cycles, but if it is
something that adds review it may take longer

**Comment by WilliamDavidHarrison** [comment]
> sounds good. there's big backlog on available cycles, but if it is something that adds review it may take longer

I have updated the submitter email: 2310270abe83684108b81a1572430e304183cfa3

File: public_suffix_list.dat [hunk 1]
```diff
@@ -14373,6 +14373,13 @@ simplesite.pl
 // Submitted by Eddie Jones <eddie@onefoldmedia.com>
 nid.io
 
+// Open Domains : https://open-domains.net
+// Submitted by William Harrison <william@williamharrison.dev>
```
Comment (simon-friedberger) [inline public_suffix_list.dat:14377]: Yes, please update.

**Comment by simon-friedberger** [comment]
* [ ] Expiration (Note: Must STAY >2y at all times)
  * [ ] is-cool.dev expires 2026-01-16
  * [x] is-not-a.dev expires 2026-08-17
  * [x] localplayer.dev expires 2026-07-18
  * [x] is-local.org expires 2026-12-03
* [x] DNS _psl entries (Note: Must STAY in place)
* [x] Tests pass
* [x] Sorting
* [x] Reasoning/Organization description

File: public_suffix_list.dat (hunk 1 above)
Comment (wdhdev) [reply public_suffix_list.dat:14377]: @simon-friedberger What do I need to update here?

**Comment by andrewstech** [comment]
> * [ ]  Expiration (Note: Must STAY >2y at all times)
>   
>   * [ ]  is-cool.dev expires 2026-01-16
>   * [x]  is-not-a.dev expires 2026-08-17
>   * [x]  localplayer.dev expires 2026-07-18
>   * [x]  is-local.org expires 2026-12-03
> * [x]  DNS _psl entries (Note: Must STAY in place)
> * [x]  Tests pass
> * [x]  Sorting
> * [x]  Reasoning/Organization description

Hi the domain has now been renewed till Jan 16, 2029
Thanks

File: public_suffix_list.dat (hunk 1 above)
Comment (simon-friedberger) [reply public_suffix_list.dat:14377]: Sorry, I misread your question here.
> @simon-friedberger Is it possible for me to update the submission email on the PR (currently [admin@open-domains.net](mailto:admin@open-domains.net)) without having our position in the queue affected?
Please update the e-mail here to one that isn't tied to one particular person.
You can also update the e-mail on the PR.

**Comment by wdhdev** [comment]
@simon-friedberger Is there anything else we need to do to get accepted?

File: public_suffix_list.dat (hunk 1 above)
Comment (wdhdev) [reply public_suffix_list.dat:14377]:
```suggestion
// Submitted by William Harrison <admin@open-domains.net>
```

**Comment by py660** [comment]
Does "Reviewed / Approved in List Add/Mod/Del" mean that it will be merged soon?

**Pull request merged**